### ISSUE TITLE
delete-build-on-push - notice ** New Issue - 

### ISSUE THREAD

**Issue by krishna-vanvlack** [VanVlackApplication/JenkinsChanges#27]
The VanVlack application did a WebHook Push and the Jenkins Server Received it. 
@krishna-vanvlack pay attention to this. 

@colossus9, @nathos, and @mattcantstop this is just a merge Webhook. 

Build 'delete-build-on-push' is failing!

Last 50 lines of build output:

```
Started by GitHub push by krishna-vanvlack
Started by GitHub push by krishna-vanvlack
[EnvInject] - Loading node environment variables.
Building in workspace /var/lib/jenkins/workspace/delete-build-on-push
 > git rev-parse --is-inside-work-tree # timeout=10
Fetching changes from the remote Git repository
 > git config remote.deleted_build.url https://github.com/VanVlackApplication/JenkinsDelete # timeout=10
Fetching upstream changes from https://github.com/VanVlackApplication/JenkinsDelete
 > git --version # timeout=10
using GIT_ASKPASS to set credentials me
 > git fetch --tags --progress https://github.com/VanVlackApplication/JenkinsDelete +refs/heads/*:refs/remotes/deleted_build/*
 > git rev-parse deleted_build/master^{commit} # timeout=10
Checking out Revision 6d92007cf23b3c24fc525a32ce5ec4afb7d6eef8 (deleted_build/master)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f 6d92007cf23b3c24fc525a32ce5ec4afb7d6eef8
Commit message: "updating for now"
 > git rev-list --no-walk 576bbf10001ee805262997b1a5293095bf2ccdeb # timeout=10
[Set GitHub commit status (universal)] PENDING on repos [GHRepository@a6397e5[description=this is the repo that is deleted,homepage=<null>,name=JenkinsDelete,license=<null>,fork=false,size=49,milestones={},language=<null>,commits={},source=<null>,parent=<null>,responseHeaderFields={null=[HTTP/1.1 200 OK], Access-Control-Allow-Origin=[*], Access-Control-Expose-Headers=[ETag, Link, Retry-After, X-GitHub-OTP, X-RateLimit-Limit, X-RateLimit-Remaining, X-RateLimit-Reset, X-OAuth-Scopes, X-Accepted-OAuth-Scopes, X-Poll-Interval], Cache-Control=[private, max-age=60, s-maxage=60], Content-Encoding=[gzip], Content-Security-Policy=[default-src 'none'], Content-Type=[application/json; charset=utf-8], Date=[Mon, 08 Jan 2018 06:41:45 GMT], ETag=[W/"b245689ed748a3d9ce4655f9c265719b"], Last-Modified=[Mon, 08 Jan 2018 06:39:06 GMT], OkHttp-Received-Millis=[1515393705317], OkHttp-Response-Source=[CONDITIONAL_CACHE 200], OkHttp-Selected-Protocol=[http/1.1], OkHttp-Sent-Millis=[1515393705256], Server=[GitHub.com], Status=[200 OK], Strict-Transport-Security=[max-age=31536000; includeSubdomains; preload], Transfer-Encoding=[chunked], Vary=[Accept, Authorization, Cookie, X-GitHub-OTP], X-Accepted-OAuth-Scopes=[repo], X-Content-Type-Options=[nosniff], X-Frame-Options=[deny], X-GitHub-Media-Type=[github.v3; format=json], X-GitHub-Request-Id=[CB42:655F:BE8A77:1AD5E7A:5A5312A8], X-OAuth-Scopes=[admin:gpg_key, admin:org, admin:org_hook, admin:public_key, admin:repo_hook, delete_repo, gist, notifications, repo, user], X-RateLimit-Limit=[5000], X-RateLimit-Remaining=[2585], X-RateLimit-Reset=[1515394741], X-Runtime-rack=[0.048085], X-XSS-Protection=[1; mode=block]},url=https://api.github.com/repos/VanVlackApplication/JenkinsDelete,id=116318287]] (sha:6d92007) with context:organization
Setting commit status on GitHub for https://github.com/VanVlackApplication/JenkinsDelete/commit/6d92007cf23b3c24fc525a32ce5ec4afb7d6eef8
[delete-build-on-push] $ /bin/sh -xe /tmp/hudson4197534796149149460.sh
+ ghi
/tmp/hudson4197534796149149460.sh: line 2: ghi: command not found
Build step 'Execute shell' marked build as failure

```

Changes since last successful build:
- [noreply] 183bc3605f32fe0460b5e60ae2c519cbeccbba31 - testing a hook

- [noreply] 7df5fde3cc7feb6cb8078cfc56e0a6bfa2eda290 - Delete updated

- [noreply] 8ce5503394346ea50942c3c6928b6ee2ab88179c - delete

- [noreply] 6d92007cf23b3c24fc525a32ce5ec4afb7d6eef8 - updating for now

[View full output](http://18.217.248.72:8080/job/delete-build-on-push/11/)
